src/test/resources/config/samples/library_0.yaml

```diff
@@ -44,3 +44,27 @@ task_templates:
   - src/main/resources/scripts/tpcds/optimize_split/spark/o_catalog_returns_IN-delta.sql
   - src/main/resources/scripts/tpcds/optimize_split/spark/o_catalog_returns_NULL-delta.sql
   custom_task_executor: com.microsoft.lst_bench.task.custom.DependentTaskExecutor
+tasks:
+# Optimize split instance of task
+- id: optimize_split_delta
+  template_id: optimize_split_delta
+  task_executor_arguments:
+    dependent_task_batch_size: 100
+    # TODO: Remove this once #182 is fixed
+    skip_erroneous_query_strings: "[DELTA_FAILED_RECOGNIZE_PREDICATE]"
+tasks_sequences:
+# Data maintenance session
+- id: seq_data_maintenance_delta
+  tasks:
+  - template_id: data_maintenance_delta
+  - template_id: data_maintenance_delta
+session_templates:
+# Data maintenance session
+- id: session_data_maintenance_delta
+  tasks:
+  - tasks_sequence_id: seq_data_maintenance_delta
```

Comment:
That's a fair point. I think this provides a lot of flexibility though it is weird because as you mention, the sequence is not a single task but rather an entity that gets expanded into multiple tasks. In that sense, it is somehow analogous to custom tasks in the sense that it can dynamically span new tasks? I'll think a bit further about this abstraction.

Reply:
I agree that it adds a lot of flexibility, I'm just worried that nesting it this way will get confusing. I don't think it needs to be put into a custom task executor but I'd prefer if we could make sure that we don't overcomplicate in our effort to provide flexibility. That's why I think we should stick with the phase > sessions > task kind of structure but allow for the session > task connection to contain either a (series of) task(s), a pre-defined sequence, or a prepared task, and not push the second into the first somehow. Does that make sense?

Reply:
I have changed this to avoid mixing tasks with sequences in the same list. However, you can still declare a sequence by inlining a list of tasks; thus, we do not lose flexibility while having a clearer schema.

Reply:
It took me a little bit but I get it now! I think some additions to the README like we discussed might be useful but LGTM otherwise.

Reply:
I have separated the documentation for the workloads from the README and started making changes. However, I think this is going to take a bit of time / can be probably reviewed on its own. For that reason, I'm checking this one in and will address the request for more documentation in #177.